### PR TITLE
Improve documentation for building plugins with Kotlin 1.5

### PR DESCRIPTION
**Pull request by wolfs**
This improves the documentation for the deprecation message when Gradle encounters a lambda as a task action with regards to Kotlin 1.5.

Fixes https://github.com/gradle/gradle/issues/17086

### CODE REVIEW

**Comment by wolfs** [comment]
@bot-gradle test QFL

File: subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/NestedInputKotlinImplementationTrackingIntegrationTest.groovy [hunk 1]
```diff
@@ -137,6 +137,62 @@ class NestedInputKotlinImplementationTrackingIntegrationTest extends AbstractPlu
         project2.file('build/tmp/myTask/output.txt').text == "hello"
     }
 
+    def "task action defined in Kotlin 1.5 can be tracked when using language versino 1.4"() {
```
Comment (eskatos) [inline subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/NestedInputKotlinImplementationTrackingIntegrationTest.groovy:140]: Typo
```suggestion
    def "task action defined in Kotlin 1.5 can be tracked when using language version 1.4"() {
```

**Review by eskatos — APPROVED** [review]
Found a typo, otherwise LGTM!
Thank you for taking care of this

**Comment by wolfs** [comment]
Thank you, @eskatos

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Pull request merged**